cluster/addons.go

```diff
@@ -38,15 +44,113 @@ func (c *Cluster) deployK8sAddOns(ctx context.Context) error {
 func (c *Cluster) deployUserAddOns(ctx context.Context) error {
 	log.Infof(ctx, "[addons] Setting up user addons..")
 	if c.Addons == "" {
-		log.Infof(ctx, "[addons] No user addons configured..")
-		return nil
+		return c.deployAddonsInclude(ctx)
 	}
 
 	if err := c.doAddonDeploy(ctx, c.Addons, UserAddonResourceName); err != nil {
 		return err
 	}
 	log.Infof(ctx, "[addons] User addon deployed successfully..")
+	return c.deployAddonsInclude(ctx)
```

Comment:
instead of returning user include here, can you add it with `if err :=` and if err is nil then we can print the success message